includes/definitions/ios.yaml

```diff
@@ -8,6 +8,15 @@ over:
     - { graph: device_processor, text: 'CPU Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
 icon: cisco
+discovery:
+    -
+      sysDescr:
+          - 'Cisco Internetwork Operating System Software'
+          - 'IOS (tm)'
+          - 'Cisco IOS Software'
+          - 'Global Site Selector'
+      sysDescr_except:
+          - 'LINUX_IOSD'
```

Comment:
Ok, went back to look through the stats data.  Not sure how I missed that, but a **few** people have C4500 and C3850 switches, which needs the IOS-XE.  Adding test data.